app/controllers/orders_controller.rb

```diff
@@ -27,8 +27,7 @@ def edit
   def create
     @order = Order.new(order_params)
     @order.add_line_items_from_cart(@cart)
-
-    @order.save!
+    @order.user_id = current_user.id
```

Comment:
can we assign user_id when builidng the order in line 28

Reply:
Yes. we can if we create new order using ```build ``` method with current user object